h2o-genmodel/src/main/java/hex/genmodel/GenModel.java

```diff
@@ -112,6 +112,10 @@ public GenModel(String[] names, String[][] domains) {
     return _domains.length - 1;
   }
 
+  @Override public String getOffsetName() {
```

Comment:
getOffsetColName ? Doesn't matter really, the var is named `_offsetColumn`, not offset.

Reply:
this is modeled to be consistent with `getResponseName`